modules/store/spec/store.spec.ts

```diff
@@ -187,7 +187,6 @@ describe('ngRx Store', () => {
 
       dispatcher.ngOnDestroy();
 
-      expect(storeSubscription.closed).toBe(true);
```

Comment:
Yeah, I'm seeing the same thing. I think this behavior changed in newer versions of rxjs.